client/src/js/app.js

```diff
@@ -757,201 +440,6 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
-  })
-  .state('/reports/bilan', {
-    controller : 'configureBilan',
-    templateUrl : 'partials/reports_proposed/bilan/bilan.html'
-  })
-  .state('/reports/debtor_group_report', {
-    controller : 'DebtorGroupReportController as debtorGroupReportCtrl',
-    templateUrl : 'partials/reports_proposed/debtor_group_report/debtor_group_report.html'
-  })
-  .state('/reports/result_account', {
-    controller : 'configureResult',
-    templateUrl : 'partials/reports_proposed/result_account/result_account.html'
-  })
-  .state('/reports/balance', {
-    controller : 'configureBalance',
-    templateUrl : 'partials/reports_proposed/balance/balance.html'
-  })
-  .state('/reports/debtorgroup/annual', {
-    controller : 'DebtorGroupAnnualReportController as AnnualCtrl',
-    templateUrl : 'partials/reports_proposed/debtor_group/annual.html'
-  })
-  .state('/reports/grand_livre', {
-    controller : 'configureGrandLivre',
-    templateUrl : 'partials/reports_proposed/grand_livre/grand_livre.html'
-  })
-  .state('/reports/employee_state', {
-    controller : 'configureEmployeeState',
-    templateUrl : 'partials/reports_proposed/employee_state/employee_state.html'
-  })
-  .state('/reports/cotisation_payment', {
-    controller : 'cotisation_payment',
-    templateUrl : 'partials/reports/cotisation_payment/cotisation_payment.html'
-  })
-  .state('/reports/salary_payment', {
-    controller : 'salary_payment',
-    templateUrl : 'partials/reports/salary_payment/salary_payment.html'
-  })
-  .state('/reports/taxes_payment', {
-    controller : 'taxes_payment',
-    templateUrl : 'partials/reports/taxes_payment/taxes_payment.html'
-  })
-  .state('/reports/stock_status', {
-    controller : 'StockStatusReportController as StatusCtrl',
-    templateUrl : 'partials/reports/stock_status/stock_status.html'
-  })
-  .state('/reports/loss_record', {
-    controller : 'loss_record',
-    templateUrl : 'partials/reports/loss_record/loss_record.html'
-  })
-  .state('/reports/income_report', {
-    controller : 'primary_cash.incomeReport',
-    templateUrl : 'partials/reports/primary_cash/income/income_report.html'
-  })
-  .state('/reports/expense_report', {
-    controller : 'primary_cash.expenseReport',
-    templateUrl : 'partials/reports/primary_cash/expense/expense_report.html'
-  })
-  .state('/reports/cash_flow/', {
-    controller : 'cashFlowReportController as ReportCtrl',
-    templateUrl : 'partials/reports/cash_flow/cash_flow.html'
-  })
-  .state('/reports/stock_report', {
-    controller : 'stock_report',
-    templateUrl : 'partials/reports/stock/stock_report.html'
-  })
-  .state('/reports/patient_visit_status', {
-    controller : 'ReportPatientVisitStatus',
-    templateUrl : '/partials/reports/patient_visit_status/patient_visit_status.html'
-  })
-  .state('/reports/stock_entry', {
-    controller : 'ReportStockEntryController',
-    templateUrl : 'partials/reports/stock/stock_entry/stock_entry.html'
-  })
   .state('/error403', {
```

Comment:
Whoops, looks like you forgot this `/`.  Could you remove it?  There should not be any slashes in the state names.